CONTRIBUTING.md

```diff
@@ -22,15 +22,17 @@ When working with Eclipse or IntelliJ please follow the [sonar guidelines](https
 ## Pull Request (PR)
 
 To submit a contribution:
-- Make sure an issue exists in this repository. Otherwise, create an issue to describe the necessity of the change.
-- Create a pull request in this repository and link it to the issue it solves (`Fixes #...`).
+- Make sure an issue exists in [SCAN4NET project](https://sonarsource.atlassian.net/browse/SCAN4NET). Otherwise, create a community thread first.
```

Comment:
is this for us or for the external users?
Seems more like it's for external users and the instruction seems kind of pointless as they cannot access S4NET jira. :/

Reply:
It's for external users. That URL provides publicly visible list of tickets.